packages/data-driver-postgres/src/index.ts

```diff
@@ -33,9 +33,9 @@ export default class DataDriverPostgres implements DataDriver {
 
 	async query(query: AbstractQuery): Promise<Readable> {
 		try {
-			const sqlStatement = convertAbstractQueryToSqlStatement(query);
-			const sqlString = constructSql(sqlStatement);
-			const queryStream = new QueryStream(sqlString /*, sqlStatement.parameters */);
+			const abstractSQLQuery = convertAbstractQueryToAbstractSqlQuery(query);
```

Comment:
Sorry for nitpicking but should it be `abstractSqlQuery` instead of `abstractSQLQuery` to match the function name?
```suggestion
			const abstractSqlQuery = convertAbstractQueryToAbstractSqlQuery(query);
```

Reply:
done ✅